caddyhttp/fastcgi/setup.go

```diff
@@ -62,8 +64,14 @@ func fastcgiParse(c *caddy.Controller) ([]Rule, error) {
 			Root: absRoot,
 			Path: args[0],
 		}
+
 		upstreams := []string{args[1]}
 
+		srvUpstream := false
+		if upstreams[0][:6] == "srv://" {
```

Comment:
Though it is a user error if `len(upstreams[0]) < 6`, it is preferred to avoid panics. `strings.HasPrefix` is a better candidate here.